### PR TITLE
enabling FIPS on amq-broker

### PR DESCRIPTION
**Pull request by aviavraham**

### CODE REVIEW

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/.config.prowgen [hunk 1]
```diff
@@ -1,2 +1,2 @@
 private: true
-expose: true
\ No newline at end of file
+expose: true
```
Comment (vi-patel) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/.config.prowgen:2]: I would revert this file so it doesn't show up as part of the PR

File: ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.17-lp-interop.yaml [hunk 2]
```diff
@@ -25,6 +25,7 @@ tests:
     cluster_profile: aws-cspi-qe
     env:
       BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      FIPS_ENABLED: "true"
```
Comment (vi-patel) [inline ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.17-lp-interop.yaml:28]: FIPS_ENABLED is added to the wrong test, it should be added to `amq-broker-interop-aws-fips`

File: ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.17-lp-interop.yaml [hunk 3]
```diff
@@ -42,6 +43,29 @@ tests:
     test:
     - ref: amq-broker-install-and-tests
     workflow: firewatch-ipi-aws
+- as: amq-broker-interop-aws-fips
+  cron: 0 6 11 2 *
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
```
Comment (dfrazzette) [inline ci-operator/config/rh-messaging-qe/claire/rh-messaging-qe-claire-lpt__amq-broker-ocp4.17-lp-interop.yaml:51]:
```suggestion:-0+0 
      FIPS_ENABLED: "true"
```

File: ci-operator/config/jboss-eap-qe/openshift-eap-tests/.config.prowgen
Comment (dfrazzette) [inline ci-operator/config/jboss-eap-qe/openshift-eap-tests/.config.prowgen:1]: No changes: file should be removed from PR.

**Comment by aviavraham** [comment]
I will create a new PR with fixes

**Pull request closed without merging**